provision/ubuntu/install.sh

```diff
@@ -48,13 +66,13 @@ cd /tmp/iproute2 && \
 make -j `getconf _NPROCESSORS_ONLN` && \
 make install
 
-wget --quiet $CLANG_URL
+wget --quiet "${CLANG_URL}"
 mkdir -p /usr/local
-tar -C /usr/local -xJf $CLANG_FILE
-ln -s /usr/local/$CLANG_DIR $CLANGROOT
-rm $CLANG_FILE
+tar -C /usr/local -xJf "${CLANG_FILE}"
+ln -s "/usr/local/${CLANG_DIR}" "${CLANG_ROOT}"
+rm ${CLANG_FILE}
 
-ln -s $CLANGROOT/bin/* /usr/local/bin/
+ln -s "${CLANG_ROOT}/bin/"* /usr/local/bin
```

Comment:
that `*` shouldn't be into the `"`?

Reply:
No, otherwise it will copy a file called "*"